crates/aliveness/src/lib.rs

```diff
@@ -0,0 +1,74 @@
+use std::{
+    net::{IpAddr, Ipv4Addr, SocketAddrV4},
+    time::Duration,
+};
+
+use aliveness::{BEACON_HEADER, BEACON_MULTICAST_GROUP, BEACON_PORT};
```

Comment:
You are right, I should probably move the types/constants. However, this would also require to move the `service_manager.rs` to the `aliveness_client` library as there would be otherwise a cyclic dependency.
It may be a good idea to drop the library aspect of the service entirely and rename it to `aliveness_service`. The current `aliveness_client` can then be renamed to `aliveness` since it then would include more than just the client functionality.

What are your thoughts about that?

Reply:
I agree, that seems like a good idea.

Reply:
Alright, done. The only thing I don't like about that is that `zbus` is now also required in the workspace despite it never being used outside of `tools/aliveness` (due to `service_manager`). Pepsi and the aliveness client only need the type definitions in `service_manager`, but since `zbus` is required there, it is also a dependency of the aliveness crate.